eng/bomgenerator/src/main/java/com/azure/tools/bomgenerator/models/BOMReport.java

```diff
@@ -9,6 +9,7 @@
 import java.io.FileWriter;
 import java.io.IOException;
 import java.io.InputStreamReader;
+import java.nio.file.Files;
```

Comment:
Was this previously a missing dependency?

Reply:
No, this is not needed any more. I was earlier does mkdirs call here but later moved it to a BomGenerator ctor.